sdk/appconfiguration/azure-spring-cloud-test-appconfiguration-config/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
```

Comment:
This test artifact is not going to be released, so we could skip the changelog content for it.

Reply:
And the version for the test module should be 1.0.0, no beta needed.